src/app/pages/MostReadPage/index.stories.jsx

```diff
@@ -33,21 +28,25 @@ export default {
 };
 
 export const Pidgin = () => (
-  <Component service="pidgin" pageData={pidginMostReadData} />
+  <Component service="pidgin" pageData={pidginMostReadData.data} />
```

Comment:
Let's just pass the page data in directly, instead of passing `.data`

```suggestion
  <Component service="pidgin" pageData={pidginMostReadData} />
```